ie_serving/server/rest_service.py

```diff
@@ -125,7 +130,19 @@ def create_rest_api(models):
     get_model_meta = GetModelMetadata(models)
     predict = Predict(models)
     app.add_route('/v1/models/{model_name}/metadata', get_model_meta)
-    app.add_route('/v1/models/{model_name}/versions/{model_version}/metadata', get_model_meta)
-    app.add_route('/v1/models/{model_name}/predict', predict)
-    app.add_route('/v1/models/{model_name}/versions/{model_version}:predict', predict)
+    app.add_route('/v1/models/{model_name}/versions/{model_version}/metadata',
+                  get_model_meta)
+    app.add_route('/v1/models/{model_name}:predict', predict)
+    app.add_route('/v1/models/{model_name}/versions/{model_version}:predict',
+                  predict)
     return app
+
+
+def start_web_rest_server(models):
```

Comment:
why this function is not included in start.py instead along with create_rest_api?